### PR TITLE
test(dataproxy,deno): test deno data proxy client

### PR DESCRIPTION
**Pull request by kt3k**
ref https://github.com/prisma/prisma/pull/15281

### CODE REVIEW

**Comment by Jolg42** [comment]
Note: tests fail with messages like 
```
The environment variable `DATABASE_URL` resolved to an empty string.
```

Because the repository secrets are not exposed when a PR is done from a fork (GitHub security default)

We would need to
- set a value for `DENO_DATA_PROXY_URL` in https://github.com/prisma/ecosystem-tests/settings/secrets/actions
- push that PR branch manually to a branch inside the repository.

**Comment by kt3k** [comment]
@millsp Could you push this branch to this repository?

I added `DENO_DATA_PROXY_URL` to `utils/crypto/envVars.ts` with the same value as other `*_DATA_PROXY_URL` values. I think now it points to the same data proxy endpoints. (So no need of addition of secrets, I guess).

**Comment by millsp** [comment]
Your PR is now running at https://github.com/prisma/ecosystem-tests/pull/3134

File: dataproxy/deno/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,33 @@
+generator client {
+  provider        = "prisma-client-js"
+  output          = "../generated/client"
+  previewFeatures = ["denoDeploy"]
```
Comment (janpio) [inline dataproxy/deno/prisma/schema.prisma:4]:
```suggestion
  previewFeatures = ["deno"]
```

File: dataproxy/deno/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "deno",
+  "license": "MIT",
+  "dependencies": {
+    "@prisma/client": "4.4.0-integration-feat-deno-deploy.1"
+  },
+  "devDependencies": {
+    "prisma": "4.4.0-integration-feat-deno-deploy.1"
```
Comment (janpio) [inline dataproxy/deno/package.json:8]: TODO

File: dataproxy/deno/run.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eu
+
+deno run -A --unstable npm:prisma@4.4.0-integration-feat-deno-deploy.1 generate --data-proxy
```
Comment (janpio) [inline dataproxy/deno/run.sh:5]: TODO

**Comment by janpio** [comment]
This will be merged via https://github.com/prisma/ecosystem-tests/pull/3134

**Pull request closed without merging**